docs/mkdocs/documentation/install.md

```diff
@@ -0,0 +1,15 @@
+# Installing
+
+## Using `kubectl`
+
+```shell
+kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/default
+```
+
+## Using the bundle
+
+### Using `operator-sdk`
```

Comment:
What is the difference between those 2?

Reply:
At some point we will add the bundle to the [Operator Hub](https://operatorhub.io/) catalog, so you will also be able to install the bundle with a `Subscription` object.